src/Apps/Auction/Components/BidForm.tsx

```diff
@@ -137,6 +152,48 @@ export const BidForm: React.FC<Props> = ({
                   {showPricingTransparency && <PricingTransparency />}
                 </Flex>
 
+                {requiresPaymentInformation && (
+                  <Box>
+                    <Separator mb={3} />
+
+                    <Serif size="4" color="black100">
+                      Please enter your credit card information below. The name
+                      on your Artsy account must match the name on the card, and
+                      a valid credit card is required in order to bid.
+                    </Serif>
+                    <Serif size="4" mt={2} color="black100">
+                      Registration is free. Artsy will never charge this card
+                      without your permission, and you are not required to use
+                      this card to pay if you win.
+                    </Serif>
+
+                    <Serif
+                      mt={4}
+                      mb={2}
+                      size="4t"
+                      weight="semibold"
+                      color="black100"
+                    >
+                      Card Information
+                    </Serif>
+
+                    <CreditCardInput
+                      error={{ message: errors.creditCard } as stripe.Error}
+                    />
+
+                    <Box mt={2}>
+                      <AddressForm
```

Comment:
Our registration form is implemented with Formik, but here we are using the AddressForm directly without Formik? Why the difference? Is there a way we can make the approach here more consistent?

Reply:
Nvm! I see now that this is within a larger Formik component.